internal/backend/gdrive/gdrive_internal_test.go

```diff
@@ -0,0 +1,201 @@
+package gdrive
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	rtest "github.com/restic/restic/internal/test"
+	drive "google.golang.org/api/drive/v3"
+)
+
+func assertNil(t *testing.T, s string) {
+	if s != "" {
+		t.Fatalf("'%s' is not nil", s)
```

Comment:
* Use `%q` for a quoted string
 * `s` is a string, which can never be `nil`, it can only be empty.